exonum-java-binding/core/rust/src/utils/errors.rs

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-use jni::objects::JObject;
-use jni::JNIEnv;
-
 use std::any::Any;
 use std::cell::Cell;
 use std::error::Error;
 use std::result;
 use std::thread;
 
+use jni::objects::JObject;
```

Comment:
fyi, we followed https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md in terms of import items order. Though I don't think it's a problem, some kind of uniformity may be useful